### PR TITLE
[MIG] partner_ref_unique: Migration to 16.0

### PR DESCRIPTION
**Pull request by deu-dev**
#1353

### CODE REVIEW

File: partner_ref_unique/tests/test_res_partner_ref.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2020 Tecnativa - João Marques
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import common
+
+
+class TestResPartnerRefUnique(common.SavepointCase):
```
Comment (AnizR) [inline partner_ref_unique/tests/test_res_partner_ref.py]: SavepointCase is deprecated. Maybe use TransactionCase?

Comment (deu-dev) [reply partner_ref_unique/tests/test_res_partner_ref.py]: @AnizR Changed

**Comment by rousseldenis** [comment]
@deu-dev Thanks for this.

Could you follow migration guide in order to keep commits history (Full process part) ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

**Comment by rousseldenis** [comment]
@deu-dev Do you plan to fix this ?

**Comment by diogocsc** [comment]
@rousseldenis I created https://github.com/OCA/partner-contact/pull/1495 with full commit history meanwhile.

**Pull request closed without merging**